### PR TITLE
Show related incidents below input

### PR DESCRIPTION
**Pull request by lmcnulty**
Addresses #868. Makes each of the the related incident views appear under the corresponding form field. Also removes the option to set similarity based on the URL or ID, as those will identify the same incident, not similar ones, which could be confusing.

image

### CODE REVIEW

**Comment by smcgregor** [comment]
I think it would be better to display the incident title and potentially description here, rather than the complete list of reports. Some reports do not have informative titles, whereas we decide the incident titles.

**Comment by lmcnulty** [comment]
The `RelatedIncidentsArea` can now display reports or incidents. Incidents are displayed under the incident id field and the text field:

image

---

image

Whereas for the fields that are attributes of the report (author, date published, and url) we still display the reports instea:

image

**Review by cesarvarela — CHANGES_REQUESTED** [review]
Looks good but it is missing translations

File: site/gatsby-site/src/components/RelatedIncidents.js [hunk 1]
```diff
@@ -55,11 +56,11 @@ const reportsWithIncidentIds = async (reports, client) => {
   }));
 };
 
-const searchColumns = {
+const allSearchColumns = {
   byDatePublished: {
     header: (incident) => (
       <>
-        Incidents reports matched by published date: <b>{incident.date_published}</b>
+        <Trans>Incidents reports matched by published date:</Trans> <b>{incident.date_published}</b>
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/RelatedIncidents.js:63]: Informes de incidentes por fecha de publicación:

(this should be only one trans component that includes the `b` tags)

File: site/gatsby-site/src/components/RelatedIncidents.js [hunk 2]
```diff
@@ -84,20 +85,24 @@ const searchColumns = {
   byIncidentId: {
     header: (incident) => (
       <>
-        Incidents reports matched by ID: <b>{incident.incident_id}</b>
+        <Trans>Incident matched by ID:</Trans> <b>{incident.incident_id}</b>
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/RelatedIncidents.js:88]: Incidentes por ID:

(this one needs to be in one Trans component too.)

File: site/gatsby-site/src/components/RelatedIncidents.js [hunk 3]
```diff
@@ -84,20 +85,24 @@ const searchColumns = {
...
+    editSimilar: false,
+    editId: false,
+    showIncidents: true,
   },
 
   byAuthors: {
     header: (incident) => (
       <>
-        Incidents reports matched by authors: <b>{incident.authors}</b>
+        <Trans>Incidents reports matched by authors:</Trans> <b>{incident.authors}</b>
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/RelatedIncidents.js:105]: Informes por autor:

(ditto)

File: site/gatsby-site/src/components/RelatedIncidents.js [hunk 4]
```diff
@@ -111,21 +116,35 @@ const searchColumns = {
   byURL: {
     header: (incident) => (
       <>
-        Incidents reports matched by URL: <b>{incident.url}</b>
+        <Trans>Incidents reports matched by URL:</Trans> <b>{incident.url}</b>
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/RelatedIncidents.js:119]: Informes por URL

(ditto)

**Review by cesarvarela — CHANGES_REQUESTED** [review]
Next time please create the i18n keys in the appropriate json and we'll comment with the translations there, so it is faster than me having to search for each <Trans> component 🍷

**Comment by lmcnulty** [comment]
Sorry about the missing keys – I would have added them but thought we had decided not to do that on account of a future system to detect missing translations automatically.

**Comment by cesarvarela** [comment]
Yea, I think I miscommunicated that, once French is added we'll have to look into the automated thing, but for now just create the empty JSONs and just ping either Clara, Pablo or me, and we'll give the translated text.

**Review by cesarvarela — APPROVED** [review]

**Pull request merged**